CHANGELOG.md

```diff
@@ -1,19 +1,20 @@
 # Change Log
 
-## Unreleased
+## 0.6.0 - 2021-04-13
 ### Added
 - Now depends on the "Azure Resources" extension, which provides a "Resource Groups" and "Help and Feedback" view
+- IntelliSense for `staticwebapp.config.json` file
```

Comment:
I would say `IntelliSense and snippets for ...`. I believe those two things are distinct

Reply:
I think IntelliSense encompasses both based off of this line:
> VS Code IntelliSense offers different types of completions, including language server suggestions, snippets, and simple word based textual completions.

https://code.visualstudio.com/docs/editor/intellisense